be/src/storage/primary_key_encoder.cpp

```diff
@@ -329,6 +348,50 @@ Status PrimaryKeyEncoder::create_column(const vectorized::Schema& schema,
     return Status::OK();
 }
 
+Status PrimaryKeyEncoder::create_column_with_sort_key(const vectorized::Schema& schema,
```

Comment:
Can you reuse create_column() to implement this function, avoid duplicated code?

Reply:
yes, changed.